utils/update_checkout/update-checkout-config.json

```diff
@@ -135,7 +135,7 @@
                 "llbuild": "main",
                 "swift-tools-support-core": "main",
                 "swiftpm": "main",
-                "swift-argument-parser": "1.0.3",
+                "swift-argument-parser": "1.1.2",
```

Comment:
The other two places where the version is mentioned need to be updated as well.

Reply:
Done 👍

Reply:
Thanks!